docs/articles_en/get-started/install-openvino/install-openvino-pip.rst

```diff
@@ -15,6 +15,19 @@ Install Intel® Distribution of OpenVINO™ Toolkit from PyPI Repository
    * is dedicated to users of all major OSes: Windows, Linux, and macOS
      (all x86_64 / arm64 architectures)
    * macOS offers support only for CPU inference
+   
+| **Simplified Build and Integration**  
```

Comment:
should it be moved to some place below after pip package is installed?
E.g. we installed package, checked that device is available (OpenVINO is functional) and then we can highlight this mode

Reply:
Yes, that would be our suggestion, as mentioned in the previous comment. We can make this change in a PR to follow, not to delay publishing.